app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,66 @@
+class ProjectsController < ApplicationController
+  before_action :set_project_and_hackday, :only => [:edit, :update, :destroy, :upvote, :downvote]
```

Comment:
seems like this is basically everything. would be better to use :except rather than :only